modules/gfo-base.owl

```diff
@@ -86,27 +83,15 @@
 
   <owl:Class rdf:about="https://w3id.org/gfo/base/Category">
     <rdfs:label xml:lang="en">Category</rdfs:label>
-    <skos:definition rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Categories satisfy the
-      following conditions: (1) Categories can be instantiated; (2) Categories can be predicated of
-      other entities.
-      Categories are defined intensional-with-an-s. They are, therefore, closely related to
-      language.</skos:definition>
+    <skos:definition rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Categories satisfy the following conditions: (1) Categories can be instantiated; (2) Categories can be predicated of other entities. Categories are defined intensional-with-an-s. They are, therefore, closely related to language.</skos:definition>
     <rdfs:subClassOf rdf:resource="https://w3id.org/gfo/base/Item" />
     <owl:disjointWith rdf:resource="https://w3id.org/gfo/base/Individual" />
   </owl:Class>
 
   <owl:Class rdf:about="https://w3id.org/gfo/base/Change">
     <rdfs:label xml:lang="en">Change</rdfs:label>
-    <skos:definition rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A change in the technical
-      sense refers to a pair of process
-      boundaries. Either at coinciding boundaries (then it comes close to
-      notions like ``punctual'' or ``instantaneous event'' as well as
-      ``moment'' -- in a temporal reading), or at boundaries at the opposite
-      ends of a process of arbitrary extension.
-
-      The notion of change is relative to contradictory conditions
-      between which a transition takes place. These contradictions refer to
-      some collection of pairwise disjoint universals.</skos:definition>
+    <skos:definition rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A change in the technical sense refers to a pair of process boundaries. Either at coinciding boundaries (then it comes close to notions like ``punctual'' or ``instantaneous event'' as well as ``moment'' -- in a temporal reading), or at boundaries at the opposite ends of a process of arbitrary extension.
+The notion of change is relative to contradictory conditions between which a transition takes place. These contradictions refer to some collection of pairwise disjoint universals.</skos:definition>
```

Comment:
There should never be a newline in a definition, because they are interpreted differently between operating systems and might interfere with programs which display the ontology later on.

Reply:
After a quick Internet search I was not able to find any example or statement which supports what I am suggesting, so its more of a personal opinion. But as stated above, avoid newlines whenever possible might help with later processing because 
* of issues, such as in https://github.com/Onto-Med/GFO/pull/34#issuecomment-2331076520 (line endings in Windows)
* changed line endings might also lead to "bloated" commit-diffs, because Git thinks something changed when in fact only the editor used the operating systems default when saving a file

However, there is no need to invest more time than necessary. Just my two cents, because you picked up my comment from https://github.com/Onto-Med/GFO/pull/42#issuecomment-2520420872